src/aap_eda/api/views/activation.py

```diff
@@ -269,6 +267,9 @@ def retrieve(self, request, pk: int):
                 description="Return a list of Activations.",
             ),
         },
+        parameters=generate_query_params(
```

Comment:
what a good point now `generate_query_params` is it? :)

Reply:
yessir, pretty dynamic. :)